.github/workflows/testing.yml

```diff
@@ -55,17 +59,14 @@ jobs:
           # dev/debug you're encouraged to start it yourself in a separate
           # terminal.
           TESTING_START_SERVER: true
-        run: |
-          # Needed for the puppeteer tests that start a static Express server,
-          # but it starts it from within the testing/ path.
-          echo "CONTENT_ROOT=content/files" >> .env
 
-          # In terms of the --maxWorkers option, it's not yet clear which
-          # is best.
-          # See https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server
-          # and https://www.peterbe.com/plog/ideal-number-of-workers-in-jest-maxWorkers
-          # CI tends to have fewer CPUs than laptops so let's stay conservative
-          # for now.
-          # Also note that `--runInBand` is the same as `--maxWorkers=1`
-
-          ./testing/scripts/functional-test.sh --runInBand
+          CONTENT_ROOT: testing/content/files
+        # In terms of the --maxWorkers option, it's not yet clear which
+        # is best.
+        # See https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server
+        # and https://www.peterbe.com/plog/ideal-number-of-workers-in-jest-maxWorkers
+        # CI tends to have fewer CPUs than laptops so let's stay conservative
+        # for now.
+        # Also note that `--runInBand` is the same as `--maxWorkers=1`
```

Comment:
These above 7 lines seem to be a comment for the `run:` but in fact, it's only supposed to be for the `yarn test:functional --runInBand` command. 

So if the `run:` does more than the 1 thing it does now, it would be make sense any more.